### ISSUE TITLE
[Feature Request]: Extend `Show R code` with an information about the reproducibility features

### ISSUE THREAD

**Issue by m7pr** [insightsengineering/teal#1245]
### Feature description

A follow-up after introduction of the lockfile in here https://github.com/insightsengineering/teal/pull/1232
Idea started in here

> So far the whole reproducibility info was displayed in the modal with the module reproducibility code. Would it be good to introduce a small text in there: "Run this code on restored env from lock file available [here]". Something like that pointing to the link to that lockfile.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

**Comment by gogonzo** [comment]
Alternatively, we could introduce a button "Download bundle" which will generate a directory (packed in zip) containing:
- renv
- .Rprofile
- .Renviron
- script.R

Opening such directory should allow script.R execution.

**Comment by m7pr** [comment]
Yes, all the `options()` set in `.Rprofile`. All the `Sys.env()` packed in `.Renvrion`. `renv::activate()` packed in `.Rprofile`

**Comment by gogonzo** [comment]
Let's not forget about this when [verified flag](https://github.com/insightsengineering/teal/issues/1354)
